Art/ReneDev01/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../ReneDev01/style.css">
+    <title>Chargement Circulaire Multicolore</title>
+</head>
+<body>
+    <div class="multicolor-loader"></div>
+
+    <script>
+        // Pour masquer le chargement après un certain temps (par exemple, 5 secondes)
+        setTimeout(function() {
+           /*  var loader = document.querySelector('.loader');
+            loader.style.display = 'none'; */
+            var multicolorLoader = document.querySelector('.multicolor-loader');
+            multicolorLoader.style.display = 'none';
+        }, 40000); // 5000 millisecondes (5 secondes)
+    </script>
+</body>
```

Comment:
# Feedback - JS not allowed
As the README.md states - the contributions should not have any JS 
Please adjust your contribution in order for your animations to be only HTML and CSS related.
The only JS modification that are authorised are the only described contribution to add in `include.js`
- [ ] Please remove any JS content on that file


# Context
```html

    <script>
        // Pour masquer le chargement après un certain temps (par exemple, 5 secondes)
        setTimeout(function() {
           /*  var loader = document.querySelector('.loader');
            loader.style.display = 'none'; */
            var multicolorLoader = document.querySelector('.multicolor-loader');
            multicolorLoader.style.display = 'none';
        }, 40000); // 5000 millisecondes (5 secondes)
    </script>
```